src/VisualStudio/Core/Next/project.json

```diff
@@ -10,7 +10,7 @@
             "version": "14.3.25407",
             "suppressParent": "all"
         },
-        "Microsoft.ServiceHub.Client": "1.0.103-rc",
+        "Microsoft.ServiceHub.Client": "0.13.78-alpha-gf674a884a9",
```

Comment:
yep. to shut up the version conflict issue when using both d14 and d15 p2p references.